Content.Server/Tabletop/TabletopSystem.cs

```diff
@@ -0,0 +1,188 @@
+                _entityManager.EntityNetManager?.SendSystemNetworkMessage(
+                    new TabletopPlayEvent(table.Uid, camera.Uid, "Chess", (274 + 64, 274)), playerSession.ConnectedClient
+                );
+            }
+        }
+
+        /**
+         * <summary>Create a map related to this entity UID, if it does not already exist.</summary>
+         * <param name="uid">The entity UID to ensure a table for.</param>
+         */
+        private void EnsureTable(EntityUid uid)
+        {
+            // We already have a table, return
+            // TODO: if tables are connected, treat them as a single entity
+            if (_gameSessions.ContainsKey(uid)) return;
+
+            // Map does not exist for this entity yet, create it and store it
+            var mapId = _mapManager.CreateMap();
+            _gameSessions.Add(uid, mapId);
+
+            // Tabletop maps do not need lighting, turn it off
+            var mapComponent = _mapManager.GetMapEntity(mapId).GetComponent<IMapComponent>();
+            mapComponent.LightingEnabled = false;
+            mapComponent.Dirty();
+
+            // TODO: don't assume chess
+            SetupChessBoard(GetMapId(uid));
+        }
+
+        #region Event handlers
+
+        // Move an entity which is dragged by the user, but check if they are allowed to do so and to these coordinates
+        private void OnTabletopMove(TabletopMoveEvent msg)
+        {
+            if (!EntityManager.TryGetEntity(msg.MovedEntity, out var movedEntity))
+            {
+                return;
+            }
+
+            // TODO: some permission system, disallow movement if you're not permitted to move the item
+
+            // Move the entity and dirty it (we use the map ID from the entity so noone can try to be funny and move the item to another map)
+            var entityCoordinates = new EntityCoordinates(_mapManager.GetMapEntityId(movedEntity.Transform.MapID), msg.Coordinates.Position);
+            movedEntity.Transform.Coordinates = entityCoordinates;
+            movedEntity.Dirty();
+        }
+
+        private void OnDraggingPlayerChangedEvent(TabletopDraggingPlayerChangedEvent msg)
+        {
+            var draggedEntity = EntityManager.GetEntity(msg.DraggedEntityUid);
+
+            if (!draggedEntity.TryGetComponent<TabletopDraggableComponent>(out var draggableComponent)) return;
+
+            draggableComponent.DraggingPlayer = msg.DraggingPlayer;
+
+            if (!draggedEntity.TryGetComponent<SpriteComponent>(out var spriteComponent)) return;
+
+            if (draggableComponent.DraggingPlayer != null)
+            {
+                spriteComponent.Scale = new Vector2(1.25f, 1.25f);
+                spriteComponent.DrawDepth = (int) DrawDepth.Items + 1;
+            }
+            else
+            {
+                spriteComponent.Scale = Vector2.One;
+                spriteComponent.DrawDepth = (int) DrawDepth.Items;
+            }
```

Comment:
See above that you should not be setting sprite stuff directly for this, use a visualizer.